drafting-utils.js

```diff
@@ -0,0 +1,66 @@
+const puppeteer = require("puppeteer");
+
+// Delay Function
+const delay = (ms) => new Promise((res) => setTimeout(res, ms));
+
+configs = {
+  BLUE_5_RED_5: "https://draftlol.dawe.gg?opts=InAiOjQ1LCJiIjo0NQ",
```

Comment:
A quick comment wouldn't hurt that explains that the numbers here tell you how many bans that team gets.